projects/debouncer/src/main.c

```diff
@@ -0,0 +1,49 @@
+#include <stdbool.h>
+
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+
+#include "soft_timer.h"

```

Comment:
The spacing of these includes is odd.

Reply:
+1 this should all be one block.

In general this kind of spacing is nice:
```c
#include "this_module_header.h"

#include <std...>

#include "other headers"
```